packages/e2e-test-utils-playwright/src/request-utils/users.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * Internal dependencies
+ */
+import type { RequestUtils } from './index';
+
+/**
+ * Create user using REST API.
+ *
+ * @param {} this RequestUtils.
+ * @param {string} username User Name.
+ * @param {string} firstName First Name.
+ * @param {string} lastName Last Name.
+ */
+async function createUser(
+	this: RequestUtils,
+	username: string,
+	firstName: string,
+	lastName: string
+) {
+	// Create User
+	// https://developer.wordpress.org/rest-api/reference/users/#create-a-user
+	await this.rest( {
+		method: 'POST',
+		path: '/wp/v2/users',
+		params: {
+			username,
+			first_name: firstName,
+			last_name: lastName,
+			email: username + '@example.com',
+			password: 'secret',
+		},
+	} );
+}
+
+/**
+ * Delete user using REST API.
+ *
+ * @param {} this RequestUtils.
+ * @param {string} username User Name.
+ */
+async function deleteUser( this: RequestUtils, username: string ) {
```

Comment:
I like the way this accepts a username, it should make much nicer to use 👍

Reply:
#42657 is now merged, so you should be able to use the existing `deleteAllUsers` function.